.azurepipelines/MuDevOpsWrapper.yml

```diff
@@ -16,8 +16,8 @@ resources:
     - repository: mu_devops
       type: github
       endpoint: microsoft
-      name: microsoft/mu_devops
-      ref: refs/tags/v6.1.0
+      name: kuqin12/mu_devops
+      ref: fix_arm64

```

Comment:
Will need to change this back to `microsoft/mu_devops` and wait for the `mu_devops` release that contains the change to move the call to `Jobs/PrGate.yml` (https://github.com/microsoft/mu_devops/pull/246/files) before completing this PR.

Reply:
Agreed. This file will not be changed with the sync yml PRs pushed through.

Reply:
Those changes are released in [6.4.1](https://github.com/microsoft/mu_devops/releases/tag/v6.4.1).

Reply:
Updated to 6.5.1.